app/uk/gov/hmrc/helptosavereminder/actors/HtsUserUpdateActor.scala

```diff
@@ -37,11 +37,9 @@ class HtsUserUpdateActor(
 
   override def receive: Receive = {
     case reminder: HtsUser => {
-
-      repository.updateNextSendDate(reminder.nino.value, reminder.nextSendDate).map {
+      repository.updateNextSendDate(reminder.nino.nino, reminder.nextSendDate).map {
```

Comment:
Please use reminder.nino.value in this file.  I think this amounts to just reverting this file

Reply:
Done